TeensyDmx.cpp

```diff
@@ -136,6 +227,9 @@ TeensyDmx::TeensyDmx(HardwareSerial& uart, RdmInit* rdm) :
     m_rdmChecksum(0),
     m_deviceLabel{0}
 {
+    Serial.begin(9600);
```

Comment:
Shouldn't be in library code, only in ino.

Reply:
Agreed, but see my answer to https://github.com/chrisstaite/TeensyDmx/pull/29#discussion_r170366623 .